libs/angular-accelerator/testing/page-header.harness.ts

```diff
@@ -18,6 +18,10 @@ export class PageHeaderHarness extends ComponentHarness {
     return await this.locatorForOptional(`[title="${title}"]`)()
   }
 
+  async getElementByPTooltip(title: string) {
```

Comment:
Why is this function called something with pTooltip but is checking for area-label?